python/slsdet/detector.py

```diff
@@ -147,6 +147,33 @@ def parameters(self, value):
             value = ut.make_string_path(value)
         self.loadParameters(value)
 
+
+    @property
+    def include(self):
```

Comment:
I'm having second thoughts about the include in python. Not sure this looks intuitive... loadParameters is maybe better on a detector object, but we can discuss. On the other hand it's consistent with `d.parameters`